packages/react-accordion/src/components/AccordionItem/useAccordionItem.ts

```diff
@@ -44,6 +47,13 @@ export const useAccordionItem = (
   state.descendants = descendants;
   state.setDescendants = setDescendants;
   state.context = useCreateAccordionItemContextValue(state);
+  const navigable = useContextSelector(AccordionContext, ctx => ctx.navigable);
+  const tabsterAttributes = getTabsterAttribute({
+    groupper: {},
+  });
```

Comment:
tldr: you can resolve this, might not be too important for now

I think that we should actively work on `tabster` to make sure that we can export meaningful hooks, since there are so many options for different parts of the tabster API to shoot yourself in the foot.

Reply:
Totally agree with that. I was itching to make a `useGroupper` or smth  on `react-tabster`